src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const treeList = list.querySelectorAll('ul');
+
+treeList.style = 'position: absolute; top: 200px; left 300px; right: 700px';
```

Comment:
to add inline styles you need to write treeList.style.cssText (now your styles doesn't apply to the document) and fix the values itself, with these styles the tree is outside of the viewport 

```suggestion
treeList.style.cssText  = 'position: absolute; top: 200px; left 300px; right: 700px';
```